decidim-core/app/cells/decidim/participatory_space_dropdown_metadata/show.erb

```diff
@@ -3,7 +3,7 @@
     <%= render :metadata %>
     <%= cell "decidim/content_blocks/menu_breadcrumb_last_activity", model, hide_participatory_space: true %>
   </div>
-  <ul class="menu-bar__secondary-dropdown__menu">
```

Comment:
If remove this class, I would expect to have this css class removed also from styles
https://github.com/octree-gva/decidim/blob/25d4c388ff09eea4a2767b7677f3c836f48dc061/decidim-core/app/packs/stylesheets/decidim/_header.scss#L418

Reply:
The class is used here
https://github.com/octree-gva/decidim/blob/25d4c388ff09eea4a2767b7677f3c836f48dc061/decidim-core/app/cells/decidim/participatory_space_dropdown_metadata/show.erb#L1